build.sbt

```diff
@@ -2051,24 +2074,18 @@ lazy val designer = (project in file("designer/server"))
     processReports,
     security,
     deploymentManagerApi,
+    componentsApi,
+    requestResponseComponentsApi,
```

Comment:
and here

Reply:
Designer and Runtime should define the API of components they are supposed to work with. And with these dependencies we express it.

Reply:
Yeah, but I thought that componentsApi is only necessary. These have some engine-specific agreement that designer shouldn't be aware of

Reply:
Yes, so do I. But I got this when we don't add it:

<img width="1638" alt="Screenshot 2024-12-16 at 23 28 35" src="https://github.com/user-attachments/assets/a7ccef24-855c-4b77-a6b7-55e95ad56f7d" />

I wonder if engine<->designer separation is as good as the models/managers one?